rules/60-input-id.rules

```diff
@@ -0,0 +1,7 @@
+# do not edit this file, it will be overwritten on update
+
+ACTION=="remove", GOTO="default_end"
+
+SUBSYSTEM=="input", ENV{ID_INPUT}=="", IMPORT{builtin}="input_id"
+
+LABEL="default_end"
```

Comment:
please rename the label "input_id_end" so it doesn't look like a copy/paste error